### PR TITLE
Implement methods for realm application creating, describing, deleting and listing

### PR DESCRIPTION
**Pull request by galxy25**
- Test.

### CODE REVIEW

File: identityClient/api.go [hunk 1]
```diff
@@ -238,3 +243,55 @@ type ToznyHostedBrokerInfoResponse struct {
...
+
+// Application wraps API level values for a (client) application of a TozID realm.
+type Application struct {
+	// Server defined unique identifier for the application
+	ID string `json:"id"`
+	// Client facing identifier for the application
+	ClientID string `json:"client_id"`
+	// Human facing name of the application
+	Name string `json:"name"`
+	// Wheter this consumer is allowed to authenticate and authorize identities
```
Comment (efabens) [inline identityClient/api.go:255]: whether

File: identityClient/identityClient.go [hunk 2]
```diff
@@ -30,6 +31,53 @@ type E3dbIdentityClient struct {
...
+	path := c.Host + identityServiceBasePath + "/" + realmResourceName + "/" + params.RealmName + "/" + applicationResourceName + "/" + params.ApplicationID
+	request, err := e3dbClients.CreateRequest("GET", path, nil)
+	if err != nil {
+		return application, err
+	}
+	err = e3dbClients.MakeSignedServiceCall(ctx, request, c.SigningKeys, c.ClientID, &application)
+	return application, err
+}
+
+// CreateRealmApplication creates a realm using the specified parameters,
```
Comment (efabens) [inline identityClient/identityClient.go:68]: I don't think this comment was updated correctly.

File: identityClient/identityClient.go [hunk 3]
```diff
@@ -30,6 +31,53 @@ type E3dbIdentityClient struct {
...
+	path := c.Host + identityServiceBasePath + "/" + realmResourceName + "/" + realmName + "/" + applicationResourceName
+	request, err := e3dbClients.CreateRequest("GET", path, nil)
+	if err != nil {
+		return applications, err
+	}
+	err = e3dbClients.MakeSignedServiceCall(ctx, request, c.SigningKeys, c.ClientID, &applications)
+	return applications, err
+}
+
+// DeleteRealmApplication deletes the realm with the specified name, returning error (if any).
```
Comment (efabens) [inline identityClient/identityClient.go:46]: Should this say deletes the realm application name?

File: identityClient/api.go (hunk 1 above)
Comment (galxy25) [reply identityClient/api.go]: gracias, fixed

File: identityClient/identityClient.go (hunk 3 above)
Comment (galxy25) [reply identityClient/identityClient.go]: yes! updated

File: identityClient/identityClient.go (hunk 2 above)
Comment (galxy25) [reply identityClient/identityClient.go]: correct 😅 , now updated

**Pull request merged**